games-action/minetest/minetest-0.4.16.ebuild

```diff
@@ -76,6 +76,7 @@ src_configure() {
 		-DENABLE_GLES=0
 		-DENABLE_LEVELDB=$(usex leveldb)
 		-DENABLE_REDIS=$(usex redis)
+		-DENABLE_POSTGRESQL=0
```

Comment:
This should really be controlled via the postgres USE flag rather than disabling out right.

Reply:
It obviously would be better to have it controlled bu a useflag, But until that day I'd keep it disabled so it isn't controlled by random().

Reply:
I'm sorry. I thought you were trying to submit a fix.

Please file a bug at https://bugs.gentoo.org/.

Reply:
Actually my PR fixes a problem and you should open a bug requesting the feature that is controlled by a postgres useflag. But fine.